examples/gno.land/r/gnoland/ghverify/contract.gno

```diff
@@ -0,0 +1,140 @@
+	verifiedResult = "OK"
+)
+
+var (
+	ownerAddress = string(std.GetOrigCaller())
+	oracle       *gnorkle.Instance
+	postHandler  postGnorkleMessageHandler
+
+	handleToAddressMap = avl.NewTree()
+	addressToHandleMap = avl.NewTree()
+)
+
+func init() {
+	oracle = gnorkle.NewInstance()
+	oracle.AddToWhitelist("", []string{ownerAddress})
+}
+
+type postGnorkleMessageHandler struct{}
+
+// Handle does post processing after a message is ingested by the oracle feed. It extracts the value to realm
+// storage and removes the feed from the oracle.
+func (h postGnorkleMessageHandler) Handle(i *gnorkle.Instance, funcType message.FuncType, feed gnorkle.Feed) {
+	if funcType != message.FuncTypeIngest {
+		return
+	}
+
+	result, _, consumable := feed.Value()
+	if !consumable {
+		return
+	}
+
+	// The value is consumable, meaning the ingestion occurred, so we can remove the feed from the oracle
+	// after saving it to realm storage.
+	defer oracle.RemoveFeed(feed.ID())
+
+	// Couldn't verify; nothing to do.
+	if result.String != verifiedResult {
+		return
+	}
+
+	feedTasks := feed.Tasks()
+	if len(feedTasks) != 1 {
+		panic("expected feed to have exactly one task")
+	}
+
+	task, ok := feedTasks[0].(*verificationTask)
+	if !ok {
+		panic("expected ghverify task")
+	}
+
+	handleToAddressMap.Set(task.githubHandle, task.gnoAddress)
+	addressToHandleMap.Set(task.gnoAddress, task.githubHandle)
+}
+
+// RequestVerification creates a new static feed with a single task that will
+// instruct an agent to verify the github handle / gno address pair.
+func RequestVerification(githubHandle string) {
+	gnoAddress := string(std.GetOrigCaller())
+	oracle.AddFeeds(
+		static.NewSingleValueFeed(
+			gnoAddress,
+			"string",
+			[]feed.Task{
+				&verificationTask{
+					gnoAddress:   gnoAddress,
+					githubHandle: githubHandle,
+				},
+			},
+		),
+	)
+}
+
+// GnorkleEntrypoint is the entrypoint to the gnorkle oracle handler.
+func GnorkleEntrypoint(message string) string {
+	return oracle.HandleMessage(message, postHandler)
+}
+
+// SetOwner transfers ownership of the contract to the given address.
+func SetOwner(owner string) {
+	if ownerAddress != string(std.GetOrigCaller()) {
+		panic("only the owner can set a new owner")
+	}
+
+	ownerAddress = owner
+
+	// In the context of this contract, the owner is the only one that can
+	// add new feeds to the oracle.
+	oracle.ClearWhitelist("")
+	oracle.AddToWhitelist("", []string{ownerAddress})
+}
+
+// GetHandleByAddress returns the github handle associated with the given gno address.
+func GetHandleByAddress(address string) string {
+	if value, ok := addressToHandleMap.Get(address); ok {
+		return value.(string)
+	}
+
+	return ""
+}
+
+// GetAddressByHandle returns the gno address associated with the given github handle.
+func GetAddressByHandle(handle string) string {
+	if value, ok := handleToAddressMap.Get(handle); ok {
+		return value.(string)
+	}
+
+	return ""
+}
```

Comment:
I'd avoid Get- for getters (as is common in Go). `HandleFromAddress` / `AddressFromHandle`?

Reply:
I thought the advice was to avoid prefixing methods with Get -- is it the same for functions?  Even in our standard library we have `GetChainID`, `GetHeight`, `GetOrigSend`, etc.